addons/udes_stock/tools/__init__.py

```diff
@@ -0,0 +1,17 @@
+"""Tools and utilties for the udes_stock addon."""
+
+from odoo.exceptions import ValidationError
+from odoo.tools.translate import _
+
+
+def get_printer_by_barcode(env, barcode):
```

Comment:
Don't think this should be here.

We don't have a dependency on the print module and if it is just a helper method for returning a printer we might as well just put it in the print module.

Reply:
This will be moved to the print module. It will go in print/tools/__init__.py.

Reply:
Cool, as this is the only commit here I'll go close the PR.